tests/components/librehardwaremonitor/conftest.py

```diff
@@ -0,0 +1,290 @@
+"""Common fixtures for the LibreHardwareMonitor tests."""
+
+from collections.abc import Generator
+from unittest.mock import AsyncMock, patch
+
+from librehardwaremonitor_api.model import (
+    LibreHardwareMonitorData,
+    LibreHardwareMonitorSensorData,
+)
+import pytest
+
+
+@pytest.fixture
+def mock_lhm_client() -> Generator[AsyncMock]:
+    """Mock a LibreHardwareMonitor client."""
+    with (
+        patch(
+            "homeassistant.components.librehardwaremonitor.config_flow.LibreHardwareMonitorClient",
+            autospec=True,
+        ) as mock_client,
+        patch(
+            "homeassistant.components.librehardwaremonitor.coordinator.LibreHardwareMonitorClient",
+            new=mock_client,
+        ),
+    ):
+        client = mock_client.return_value
+        client.get_data.return_value = LHM_SAMPLE_DATA
+
+        yield client
+
+
+LHM_SAMPLE_DATA = LibreHardwareMonitorData(
```

Comment:
How do generally map this from JSON? If you can do the same here you could use a json fixture file